sandbox/settings.py

```diff
@@ -312,6 +312,29 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
         default="richie", environ_name="RICHIE_ES_INDICES_PREFIX", environ_prefix=None
     )
 
+    # LTI Content
+    RICHIE_LTI_PROVIDERS = {
+        "lti_provider_test": {
+            "oauth_consumer_key": values.Value(
+                "InsecureOauthConsumerKey",
+                environ_name="LTI_TEST_OAUTH_CONSUMER_KEY",
+                environ_prefix=None,
+            ),
+            "shared_secret": values.Value(
+                "InsecureSharedSecret",
+                environ_name="LTI_TEST_SHARED_SECRET",
+                environ_prefix=None,
+            ),
+            "base_url": values.Value(
+                environ_name="LTI_TEST_BASE_URL", environ_prefix=None
```

Comment:
Why not put the LTI test url here as default and remove your configurations in env files and in the CI? This would be more coherent with what you do for consumer and secret keys.

Reply:
It seemed strange to me to to have localhost:port by default, but if you're ok with that, I'll put it here :)

Reply:
Aaah no sorry I didn't see. I meant to put the LTI demo site here.